proto/frequenz/dispatch/v1/dispatch.proto

```diff
@@ -75,6 +75,52 @@ message Dispatch {
 
   // The dispatch payload
   google.protobuf.Struct payload = 11;
+
+  // The recurrence rule
+  RecurrenceRule recurrence_rule = 12;
+}
+
+// Information about the schedule for reoccurring dispatches
```

Comment:
I think this should provide some examples, and clarifications about the interaction between the different fields. For example, if I use `FREQUENCY_HOURLY` can I still use `by_yeardays`?

Reply:
Hmm, that is a good point. I wonder if there is a way to modify the structure so that each frequency type is paired with whichever reoccurrence fields applies to it... I'll see if I can come up with something.